tools/mason/MasonPublish.chpl

```diff
@@ -560,6 +576,32 @@ proc check(username : string, path : string, trueIfLocal : bool, ci : bool) thro
   exit(0);
 }
 
+/* Validate license with that of SPDX list */
+private proc checkLicense(projectHome: string) throws {
+  var foundValidLicense = false;
+  if exists(joinPath(projectHome, "Mason.toml")) {
+    var defaultLicense = "None";
+    const toParse = open(joinPath(projectHome, "Mason.toml"), iomode.r);
+    const tomlFile = owned.create(parseToml(toParse));
+    if tomlFile.pathExists("brick.license") {
+      defaultLicense = tomlFile["brick"]!["license"]!.s;
+    }
+    // git clone the SPDX repo and validate license identifier
+    const dest = MASON_HOME + '/spdx';
+    const command = 'git clone https://github.com/spdx/license-list ' + dest;
```

Comment:
Rationale: Shallow to reduce clone time. Specific release to prevent SPDX version we check against getting automatically updated underneath us and breaking compatibility with existing licenses.

Reply:
@ben-albrecht  Okay, I dont think we need a shallow clone here. This is actually an archived repository and it contains the latest version of SPDX licenses. It consist of only the text files of each license, so its very small and clone time is very less.

Reply:
Yeah, I see a different of ~0.15 seconds for shallow vs. full clone. It wouldn't hurt to do a shallow clone, but it's also fine to leave as is.